### PR TITLE
add: create tests for Tabs and Tab components

### PR DESCRIPTION
**Pull request by HuziMuzi**
create tests for Tabs and Tab components

### CODE REVIEW

File: src/components/Tabs/Tab/__tests__/tab.test.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {fireEvent, render, screen} from '@testing-library/react-native';
+import {Tab, TabPropsT} from '../index';
+
+const onPressMock = jest.fn();
+const createTestProps = (): TabPropsT => ({
```
Comment (antoniosk10) [inline src/components/Tabs/Tab/__tests__/tab.test.tsx:5]: why you need this function?

File: src/components/Tabs/Tab/__tests__/tab.test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  item: 'test title',
+  index: 1,
+  isActive: true,
+  setCurrentTab: onPressMock,
+});
+
+describe('Tab component', () => {
```
Comment (antoniosk10) [inline src/components/Tabs/Tab/__tests__/tab.test.tsx:12]: 'Tab component functionality'

File: src/components/Tabs/Tab/__tests__/tab.test.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const props = createTestProps();
+  render(<Tab {...props} />);
+
+  test('click Tab', async () => {
```
Comment (antoniosk10) [inline src/components/Tabs/Tab/__tests__/tab.test.tsx:16]: 'Should call method on press'

File: src/components/Tabs/Tab/index.tsx [hunk 4]
```diff
@@ -1,7 +1,6 @@
-import {View} from 'react-native';
 import {ButtonTab, TitleTab} from './styles';
 
-type TabPropsT = {
+export type TabPropsT = {
```
Comment (antoniosk10) [inline src/components/Tabs/Tab/index.tsx:3]: just `TabProps` or `TabPropsType`

File: src/components/Tabs/__tests__/tabs.test.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import {render, screen} from '@testing-library/react-native';
+import {Tabs} from '../index';
+
+test('Tabs component', async () => {
```
Comment (antoniosk10) [inline src/components/Tabs/__tests__/tabs.test.tsx:4]: the same, describe more concrete what are you expecting from this test. Start from "Should...."

File: src/components/Tabs/Tab/__tests__/tab.test.tsx (hunk 1 above)
Comment (HuziMuzi) [reply src/components/Tabs/Tab/__tests__/tab.test.tsx]: She's useless

**Review by Julia-Kovalchuk — APPROVED** [review]

**Review by KhaninPasha0488 — APPROVED** [review]

**Pull request merged**